src/Components/Server/src/Circuits/CircuitHost.cs

```diff
@@ -394,6 +512,28 @@ public static void UnhandledExceptionInvokingCircuitHandler(ILogger logger, Circ
 
             public static void CircuitClosed(ILogger logger, string circuitId) =>
                 _onCircuitClosed(logger, circuitId, null);
+
+            public static void EndInvokeDispatchException(ILogger logger, Exception ex) => _endInvokeDispatchException(logger, ex);
+
+            public static void EndInvokeJSFailed(ILogger logger, long asyncHandle, string arguments) => _endInvokeJSFailed(logger, asyncHandle, arguments, null);
+
+            public static void EndInvokeJSSucceeded(ILogger logger, long asyncCall) => _endInvokeJSSucceeded(logger, asyncCall, null);
+
+            public static void DispatchEventFailedToParseEventDescriptor(ILogger logger, Exception ex) => _dispatchEventFailedToParseEventDescriptor(logger, ex);
+
+            public static void DispatchEventFailedToDispatchEvent(ILogger logger, Exception ex) => _dispatchEventFailedToDispatchEvent(logger, ex);
+
+            public static void BeginInvokeDotNet(ILogger logger, string callId, string assemblyName, string methodIdentifier, long dotNetObjectId)
```

Comment:
Just a minor thought: Since there's always exactly one out of `assemblyName` and `dotNetObjectId`, it would be reasonable to have two different methods here and avoid the `if`/`else`.

Obviously this is all an internal implementation detail - I don't really mind whether you change it or not 😄

Reply:
Id rather encapsulate the logic inside the log method rather than outside. On the CircuitHost I care about logging the call, not whether if is for an instance or a static method, does it make sense?

Reply:
I strongly don't feel strongly! Please do whatever you prefer.